e2e/autoscript-apps/src/authn-oauth/autoscript.ts

```diff
@@ -130,6 +133,28 @@ function autoscript() {
         },
       ),
       rxDelay(delay),
+      mergeMap(
+        (tokens) => {
+          console.log('Retrieve a resource');
+          return forgerock.HttpClient.request({
```

Comment:
Can we pull this out of the `authn-oauth` autoscript and test suite and create a new pair of autoscripts and suites for testing `HttpClient` specifically, or, better yet, the user-flow that it represents? My concern is that we are technically testing the `HttpClient` module. Now, we may be also testing modules that `HttpClient` pulls in internally. But, if we considered all the individual modules that something we are testing pulls in, our naming would be incredibly hard to parse.

For most testing, we should name it according to what we are directly importing and calling in our tests. For E2E, I like to refer to the test by the user-flow that it's intending to automate. The `HttpClient` isn't really directly related to authentication or OAuth, even if it uses shared modules with that functionality. So, it being included in `authn-oauth` is not semantically correct.

This is why we have autoscripts and suites for `authz-txn-basic` and such as that represents the flow we are testing. Can we name these more along those lines? It could be something as simple as `protected-request` or `authz-token` ... something describing the flow. I wouldn't be too specific though, like including "expiry" or "expiry-time" in the name as that would be too limiting.

Hit me up if you want to chat about it. Naming is ridiculously hard.